api/hypershift/v1beta1/hostedcluster_conditions.go

```diff
@@ -251,6 +251,12 @@ const (
 	RecoveryFinishedReason = "RecoveryFinished"
 
 	CloudResourcesCleanupSkippedReason = "CloudResourcesCleanupSkipped"
+
+	ControlPlaneToDataplaneReasonNoKonnectivityAgentPodsFound = "NoKonnectivityAgentPodsFound"
```

Comment:
```suggestion
	ControlPlaneToDataPlaneReasonNoKonnectivityAgentPodsFound = "NoKonnectivityAgentPodsFound"
```
(this would also need update in other places where this is used.

Reply:
Same, in a follow-up